### DISCUSSION TITLE
plantiKube integration with LearnCard

### DISCUSSION THREAD

**Discussion by nargetdev** [learningeconomy/LearnCard#298] [Show and tell]
.. a very nascent discussion regarding LearnCard use case within `plantiKube` framework - peer-to-peer modular hydroponics programmable infrastructure for nourishment and education.

((
pretext: It's nascent because GraphQL schemas, k8s manifests, live webcam timelapse stream ingresses, and digital twins for each plant grown stored in the `plantiGraph` repos are still being prepared... but will be public/live AGPL code soon at [plantiKube](https://github.com/plantiKube)
))

Now, on to the story ..

Let’s say I’m a grower.  Or an owner/steward of some physical hydroponics gear.  I’ve been growing tomatoes and arugula successfully on the infra for 2 years, integrating with some local restaurants and recurrent social events (i.e. gatherings), and now I want to pass the baton to Sally.. when Sally has been successfully growing food with the infrastructure for some time there is an integral and sound digital trail..

seeing as our `plantiGraph` stores photos, Timelapses (and derived data objects via CV such as plant height, volume, health, color, etc) time series datasets on pH, electrical conductivity (nutrients in water), H20 temp, air temp, output produce weights in kg, many other metrics… these can all serve as peer-to-peer attestations of competency.

Feels like there are multiple players // agents // stakeholders involved in the programmable community agriculture sphere who can attest to one another's aptitudes via VC (verifiable credential) and assign the accreditation.

Sparking this conversation here.  Knowing it's half baked. My prayer is that conceptual seeds will germinate and sprout {{figuratively&&literally}}

I'll update here as those network endpoints on our grow processes become live in our warehouse in RiNo Denver, as well as in the hands of users publishing observable processes.

**Comment by Custard7** [comment]
@nargetdev - thanks for sharing your project! Sounds super interesting. It's a great use case - and super cool. 🌱 

Would love to hear how the project develops! Am very curious who the players // agents // stakeholders are in the programmable community ag sphere.

Let us know if you run into anything you need help with 💪
